sql/moz-fx-data-shared-prod/firefox_accounts/fxa_all_events/view.sql

```diff
@@ -174,11 +186,27 @@ SELECT
   JSON_VALUE(event_properties, "$.email_service") AS email_service,
   JSON_VALUE(event_properties, "$.email_template") AS email_template,
   JSON_VALUE(event_properties, "$.email_version") AS email_version,
+  JSON_VALUE(event_properties, "$.subscription_id") AS subscription_id,
   JSON_VALUE(event_properties, "$.plan_id") AS plan_id,
+  JSON_VALUE(event_properties, "$.previous_plan_id") AS previous_plan_id,
+  JSON_VALUE(event_properties, "$.subscribed_plan_ids") AS subscribed_plan_ids,
```

Comment:
@biancadanforth while we originally wrote this to use `SPLIT()` to convert the comma-separated list into a proper BigQuery array, I forgot that array columns aren't easily accessible in Looker, so I changed this back to just being a string.  To determine cases when they're subscribed to multiple plans you could look for embedded commas, or to determine the number of plans they're subscribed to you could count the embedded commas and add one.

Reply:
Hmm okay. I wonder if we can create a derived dimension for this field in LookML? @ksiegler1 , would this be possible?